amppkg.example.toml

```diff
@@ -58,6 +58,10 @@ KeyFile = './pems/privkey.pem'
 # locking; consider this especially when utilizing network-mounted storage.
 OCSPCache = '/tmp/amppkg-ocsp'
 
+# If a PublicDir is specified, amppkg will look in this directory for
+# valid AMPs to package (instead of fetching AMPs from the network).
+# PublicDir = "public"
```

Comment:
Can you also explain what the path under that directory will be? Will it include the domain of the fetch URL or just its path?

Also, if it's just path -- is it possible to include the fetch domain in the disk path? It would be nice if this feature intersected well with the feature that you can have multiple URLSets.

Reply:
I've improved the comment.

I also had a look at including the fetch domain, but I can't see a good way to do this easily/cleanly... (Conceptually maybe the right place to put this is even in the `Scheme` key of `[URLSet.Fetch]`: if it's `file`, then we fetch from the local filesystem.)

But I hope it can be included as is! It's really only suitable for small test projects since it doesn't allow headers to be controlled, doesn't handle query strings, etc. (But, it does make these rather easier to deploy and reason about.)

Reply:
OK, that's fine; can be included as-is. I guess it might be possible if you implemented a custom RoundTripper that delegated to FileTransport with a modified path? Dunno, not a big deal either way.